example/pubspec.lock

```diff
@@ -180,7 +180,7 @@ packages:
       path: ".."
       relative: true
     source: path
-    version: "0.0.21"
+    version: "0.0.22"
```

Comment:
This was missed in #40.

Reply:
dang! I'm taking note to run `pub get` before committing next time.